src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_FIGURE_TYPES = 5; //5 figure types
```

Comment:
Naming is fine so code is clear so comment is excessive
```suggestion
    private static final int MAX_FIGURE_TYPES = 5;
```